examples/cms-datocms/postcss.config.js

```diff
@@ -1,18 +1,3 @@
 module.exports = {
-  plugins: [
-    'tailwindcss',
-    process.env.NODE_ENV === 'production'
-      ? [
-          '@fullhuman/postcss-purgecss',
-          {
-            content: [
-              './pages/**/*.{js,jsx,ts,tsx}',
-              './components/**/*.{js,jsx,ts,tsx}',
-            ],
-            defaultExtractor: content => content.match(/[\w-/:]+(?<!:)/g) || [],
-          },
-        ]
-      : undefined,
-    'postcss-preset-env',
-  ],
+  plugins: ['tailwindcss', 'postcss-preset-env'],
```

Comment:
Could this please be upgraded to the entire default set used by Next.js?

Reply:
@Timer AFAIK, this is the default set? I double checked the [Tailwind example config](https://github.com/zeit/next.js/blob/canary/examples/with-tailwindcss/postcss.config.js), and it matches what's there too.

I can def update, just not 100% sure where to find the entire default set?

Reply:
https://nextjs.org/docs/advanced-features/customizing-postcss-config#customizing-plugins